robottelo/vm.py

```diff
@@ -16,7 +16,7 @@
 from robottelo.common import conf, ssh
 
 
-BASE_IMAGES = ('rhel7', 'rhel65', 'rhel66')
+BASE_IMAGES = ('rhel71', 'rhel70', 'rhel66', 'rhel65')
```

Comment:
Wait, are you asking about the support for installer or support for content hosts?
for installer - currently for sat 6.1, we support only rhel66, rhel71
for content hosts - we support all, that is - rhel 5.7+, rhel6+, rhel7+
please ping me in irc if you have more questions on this